deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/daemonset.yaml

```diff
@@ -16,14 +16,18 @@ spec:
         {{- toYaml . | nindent 8 }}
     {{- end }}
       labels:
-    {{- include "harvester-network-controller.selectorLabels" . | nindent 8 }}
+        {{- include "harvester-network-controller.selectorLabels" . | nindent 8 }}
     spec:
       serviceAccountName: {{ include "harvester-network-controller.name" . }}
       hostNetwork: true
       containers:
         - name: network
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - harvester-network-controller
+          args:
+            - agent
           securityContext:
             privileged: true
           volumeMounts:
```

Comment:
The volume mounts are different from https://github.com/rancher/harvester-network-controller/blob/7dbfc4eaca8bda37e3a0adf359856f681e4a70aa/manifests/daemonset.yaml#L46